axelrod/tests/strategies/test_cycler.py

```diff
@@ -12,6 +12,7 @@
 from .test_player import TestPlayer
 
 C, D = Action.C, Action.D
+random = axl.RandomGenerator()
```

Comment:
Why are we doing this instead of using the runtime `_module_random` (created in `axelrod/__init__.py`) which is what is used in `test_filtering.py`?

Reply:
Basically I don't want to use `_module_random` if at all possible. Since I worked on the PR over quite some time, it's probably just inconsistent application of that principle. I'll look into it.

Reply:
Cool, I thought it might be something like that. I think I didn't comment on all of them.